packages/fast-components-react-base/src/dialog/dialog.props.ts

```diff
@@ -40,13 +40,20 @@ export interface IDialogHandledProps {
      */
     modal?: boolean;
 
+    /**
+     * The onDismiss callback
+     * Callback is registered on click of the modal or when escape key is pressed
```

Comment:
"...on click of modal" means when modal overlay is clicked, correct?

Reply:
yes. Updated with more detail.